### PR TITLE
Analysis options, fixes and format

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

File: place_tracker/lib/place_details.dart [hunk 1]
```diff
@@ -54,23 +54,23 @@ class PlaceDetailsState extends State<PlaceDetails> {
       children: <Widget>[
```
Comment (RedBrogdon) [inline place_tracker/lib/place_details.dart]: Once that linter is built, I'm going buck wild on these. 😄

File: place_tracker/lib/place_list.dart [hunk 2]
```diff
@@ -22,8 +22,7 @@ class PlaceListState extends State<PlaceList> {
   void _onPlaceChanged(Place value) {
     // Replace the place with the modified version.
     final List<Place> newPlaces = List.from(AppState.of(context).places);
-    final int index =
-        newPlaces.indexWhere((Place place) => place.id == value.id);
+    final int index = newPlaces.indexWhere((place) => place.id == value.id);
```
Comment (RedBrogdon) [inline place_tracker/lib/place_list.dart]: nit: Could lose the type annotations here and on 24?

File: place_tracker/lib/place_map.dart [hunk 3]
```diff
@@ -109,8 +109,7 @@ class PlaceMapState extends State<PlaceMap> {
   void _onPlaceChanged(Place value) {
     // Replace the place with the modified version.
     final List<Place> newPlaces = List.from(AppState.of(context).places);
-    final int index =
-        newPlaces.indexWhere((Place place) => place.id == value.id);
+    final int index = newPlaces.indexWhere((place) => place.id == value.id);
```
Comment (RedBrogdon) [inline place_tracker/lib/place_map.dart]: nit: remove type annotations here as well?

File: place_tracker/analysis_options.yaml [hunk 4]
```diff
@@ -1,10 +1,30 @@
...
+    - non_constant_identifier_names
+    - package_api_docs
+    - package_names
+    - package_prefixed_library_names
+    - test_types_in_equals
+    - throw_in_finally
+    - unnecessary_brace_in_string_interps
+    - unnecessary_getters_setters
+    - unnecessary_new
+    - unnecessary_statements
```
Comment (RedBrogdon) [inline place_tracker/analysis_options.yaml]: nit: newline

File: place_tracker/lib/place_details.dart (hunk 1 above)
Comment (domesticmouse) [reply place_tracker/lib/place_details.dart]: SGTM =)

File: place_tracker/lib/place_list.dart (hunk 2 above)
Comment (domesticmouse) [reply place_tracker/lib/place_list.dart]: done

File: place_tracker/lib/place_map.dart (hunk 3 above)
Comment (domesticmouse) [reply place_tracker/lib/place_map.dart]: done

File: place_tracker/analysis_options.yaml (hunk 4 above)
Comment (domesticmouse) [reply place_tracker/analysis_options.yaml]: done

**Pull request merged**